language/stdlib/modules/Addresses.move

```diff
@@ -0,0 +1,29 @@
+address 0x0 {
+module CoreAddresses {
```

Comment:
Nit: rename either the file or the module?

Reply:
Woops...forgot to rename the file after renaming the module....good spot!